modular/metadata/metadata_sp_exit_service.go

```diff
@@ -59,10 +59,6 @@ func (r *MetadataModular) GfSpGetGlobalVirtualGroupByGvgID(ctx context.Context,
 	}
 
 	if gvg != nil {
-		if err != nil {
-			log.CtxErrorw(ctx, "failed to parse global virtual group ids", "error", err)
-			return nil, err
-		}
```

Comment:
can we continue execution here with error not nil?

Reply:
yep, this is just redundant code by mistake
<img width="956" alt="image" src="https://github.com/bnb-chain/greenfield-storage-provider/assets/122767193/93a6425b-3103-4b04-a619-7d51691834f4">